.github/workflows/trigger-repository-archival.yaml

```diff
@@ -2,37 +2,22 @@ name: Trigger Unowned Repository Scan
 
 on:
   schedule:
-    - cron: "0 9 * * *"
-  workflow_dispatch:
-    inputs:
-      dry-run:
-        description: Whether to dry-run archival
-        type: boolean
-        required: true
-        default: true
```

Comment:
Why no `dispatch`? We never want to trigger this manually again?

Reply:
Hmm, I guess we might. I just opted to simplify the workflow as much as possible. I can re-allow dispatching manually if we think it's useful.